easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb

```diff
@@ -0,0 +1,1061 @@
+    }),
+    ('R.filesets', '2.14.0', {
+        'checksums': ['707dd7c099aeadbecebbd2ba57e6400405aa44487c641f3729ce06c4c642bb23'],
+    }),
+    ('aroma.light', '3.22.0', {
+        'checksums': ['c50f8be7d7b6bafca440fdd54c957ac0c288904fd69115e996f13274033578fb'],
+    }),
+    ('PSCBS', '0.65.0', {
+        'checksums': ['3365065d5375c599eb024bfff12c5f6b10a6b1a4fe4ba6f200f7e83618dd399a'],
+    }),
+    ('aroma.core', '3.2.2', {
+        'checksums': ['35e0312e403a1894da036afac12d1d727c937e666d7eb400bfe4018b21244855'],
+    }),
+    ('R.huge', '0.9.0', {
+        'checksums': ['f12d113a965603cfdbbb46e1a57331d267916235555b22f2bbc06caa302ae58e'],
+    }),
+    ('aroma.apd', '0.6.0', {
+        'checksums': ['87513774baeed8e1d43f819e615c85278aef748ebc842f1ee50686c3392e37d1'],
+    }),
+    ('aroma.affymetrix', '3.2.0', {
+        'checksums': ['d279cf13cf6ed8369cfdf6682ace12924379bad578003453449aa1ccec31b0a7'],
+    }),
+    ('Rsolnp', '1.16', {
+        'checksums': ['3142776062beb8e2b45cdbc4fe6e5446b6d33505253d79f2890fe4178d9cf670'],
+    }),
+    ('Repitools', '1.38.0', {
+        'checksums': ['7464394ebd039cd1d57e242ddb71bf02fa35a6d0dc9aafae04c834e021c36a50'],
+    }),
+    ('BSgenome.Hsapiens.UCSC.hg38', '1.4.3', {
+        'checksums': ['c29911ca93935e3b9aa6a67e894337dc2ad707405da1500c9898bd0578519ecc'],
+    }),
+    ('MEDIPS', '1.44.0', {
+        'checksums': ['3128cdc286275faffc792c4a6cf08562cad072be6b9c8595d52353f06151a750'],
+    }),
+    ('RProtoBufLib', '2.4.0', {
+        'patches': ['%(name)s-%(version)s_fix-protobuf-build.patch'],
+        'checksums': [
+            'cb2e6eba5fabea2f3ecc9a97ae25d54f2b5ad14ed08381a83db612af4ed2cdc3',  # RProtoBufLib_2.4.0.tar.gz
+            # RProtoBufLib-2.4.0_fix-protobuf-build.patch
+            '379e0227b22ee027de3ab22be9667f358de7d176f74aba598e5a3cbf41dc99a3',
+        ],
+    }),
+    ('cytolib', '2.4.0', {
+        'checksums': ['db92fc4dc0f0e8546f692b07eb1c1ab12f568705e46b87bf6b6844b039fe8772'],
+    }),
+    ('flowCore', '2.4.0', {
+        'checksums': ['cc92e084d693928e034bcda4f844eb9545589b5fc6c77637ac3d8f12fb5cb78f'],
+    }),
+    ('mutoss', '0.1-12', {
+        'checksums': ['2889ae3d502157592697124eb86adc14911e2b7fdaa7204743a376b1eeb967fa'],
+    }),
+    ('mathjaxr', '1.4-0', {
+        'checksums': ['ba57378236d593a39c5839054adc5473526de0c8f05b7eeb87c99438496ddc67'],
+    }),
+    ('metap', '1.4', {
+        'checksums': ['5fac23d823d0ad4eebc3f97620364e25f7b41f8d0c3579f6c09ec059940b85a5'],
+    }),
+    ('uwot', '0.1.10', {
+        'checksums': ['6ee1b6027bce679cd5a35f647f516a5b327632234bcf323c7f3d5b5e10807d23'],
+    }),
+    ('scattermore', '0.7', {
+        'checksums': ['f36280197b8476314d6ce81a51c4ae737180b180204043d2937bc25bf3a5dfa2'],
+    }),
+    ('sctransform', '0.3.2', {
+        'checksums': ['5dbb0a045e514c19f51bbe11c2dba0b72dca1942d6eb044c36b0538b443475dc'],
+    }),
+    ('SeuratObject', '4.0.1', {
```

Comment:
@branfosj could you bump this to `4.0.2` please.

Reply:
This will also require `Matrix` version `1.3-4`, currently in `R`.

Reply:
done in 065f00c